docusaurus/.gitignore

```diff
@@ -0,0 +1,12 @@
+.DS_Store
+
+node_modules
+
+lib/core/metadata.js
+lib/core/MetadataBlog.js
+
+website/translated_docs
+website/build/
+website/yarn.lock
```

Comment:
Do we want to ignore the yarn.lock? Isn't that the node equivalent of ignoring the go.sum?

Reply:
good spot, that was the default gitignore from docusaurus. we should definitely keep the lock file.